components/brave_sync/brave_sync_prefs.h

```diff
@@ -72,8 +72,13 @@ extern const char kSyncMigrateBookmarksVersion[];
 extern const char kSyncRecordsToResend[];
 // Meta info of kSyncRecordsToResend
 extern const char kSyncRecordsToResendMeta[];
-// Flag indicates we had recovered duplicated bookmarks object ids
-extern const char kDuplicatedBookmarksRecovered[];
+// Flag indicates we had recovered duplicated bookmarks object ids (deprecated)
+// extern const char kDuplicatedBookmarksRecovered[];  // deprecated
```

Comment:
we should also remove this prefs from user profile if it exists

Reply:
this is fixed in 7007cb15edfab4c5ed2c96f8eae3c99d8bc68088